packages/api-derive/test/e2e/rx.spec.ts

```diff
@@ -3,15 +3,20 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import BN from 'bn.js';
+
 import ApiRx from '@polkadot/api/rx/Api';
 import { ApiInterface$Rx } from '@polkadot/api/types';
-import { BlockNumber } from '@polkadot/types';
+import { AccountId, AccountIndex, Balance, BlockNumber } from '@polkadot/types';
 import { WsProvider } from '@polkadot/rpc-provider';
 
 const WS_LOCAL = 'ws://127.0.0.1:9944/';
 // const WS_POC3 = 'wss://poc3-rpc.polkadot.io/';
 
-describe.skip('derive e2e', () => {
+// Dev account Alice
+const ID = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+const IX = 'F7Hs';
+
+describe('derive e2e', () => {
```

Comment:
`.skip`

Reply:
That's what I was wondering: Why are we skipping all those e2e tests?

Reply:
https://github.com/polkadot-js/api/pull/873#pullrequestreview-234408802

Reply:
Ah, ok, makes sense. The Alex tests failed yesterday because Alex was down again. 
I was thinking about duplicating the tests anyways (only for derive to start with), one for the latest master/ localhost and one for Polkadot/ Alexander, so we can be sure that the API works with both.